modelcheck/src/main/kotlin/Main.kt

```diff
@@ -80,7 +85,70 @@ fun printResult(item: IssueKindReportItem, project: Project, args: ModelCheckArg
     }
 }
 
-fun modelCheckProject(args: ModelCheckArgs, project: Project) : Boolean {
+
+fun writeJuniXml(models: Iterable<SModel>, results: Iterable<IssueKindReportItem>, project: Project, file: File) {
```

Comment:
Small typo here. Should be writeJuni**t**Xml I guess.

Reply:
```suggestion
fun writeJunitXml(models: Iterable<SModel>, results: Iterable<IssueKindReportItem>, project: Project, file: File) {
```

Github has this awesome feature to suggest changes to the pr. Its super useful for typos :)

Reply:
Sorry did not know about this! Will use it next time 👍